### PR TITLE
Continuous deployment to azure

### PR DESCRIPTION
**Pull request by CalPinSW**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,5 +76,5 @@ The frontend and backend applications are deployed to two Azure Web Applications
 1. Building new Docker images with (from the relevant frontend/backend directory) `docker build --target production --tag calpin/playlist-manager-"frontend/backend":prod .`
 2. Pushing the new image to Docker Hub `docker push calpin/playlist-manager-"frontend/backend":prod`
 3. Sending a curl request to the appropriate Webhook url:
-    - Frontend: `curl -dH -X POST "https://\$PlayMan:d17p8pbD0t6xpCdep4tPdetcbsTlZiomZ6kwAwuJB9XaXwjE5hmmgSpBTYqQ@playman.scm.azurewebsites.net/api/registry/webhook"`
-    - Backend: `curl -dH -X POST "https://\$PlayManBackend:aBPc2NHMrYLEHQtmPB1JhQKgtsonxezPjliwxptyx2142STiv91aRxmnmKe6@playmanbackend.scm.azurewebsites.net/api/registry/webhook"`
+    - Frontend: `curl -dH -X POST "Frontend Webhook from Azure"`
```
Comment (JackMead) [inline README.md:79]: Ah, caught! But I think not rotated in Azure still!

**Review by JackMead — APPROVED** [review]
Nice work, I can see the app running happily in Azure with the favicons, and the pipeline reporting successful deployment - go ahead and merge when you're ready!

File: README.md (hunk 1 above)
Comment (CalPinSW) [reply README.md:79]: Should now be rotated 😬

Comment (CalPinSW) [reply README.md:79]: Let's merge and see what happens 😀

**Pull request merged**